dataplane/src/main.rs

```diff
@@ -1,6 +1,120 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright Open Network Fabric Authors
+
```

Comment:
@daniel-noland I guess this line was changed by mistake, please restore the copyright notice.

Reply:
Addressed in #214